diff -rupN 6SV1.1/Makefile /Users/danclewley/Desktop/6SV1.1/Makefile
--- 6SV1.1/Makefile	2007-04-19 13:16:12.000000000 -0700
+++ /Users/danclewley/Desktop/6SV1.1/Makefile	2014-05-18 16:30:08.000000000 -0700
@@ -3,9 +3,9 @@ SYSTEM  = HP-UX
 EXTRA   = +FPVZOUD +O3 -D$(SYSTEM)
 EXTRA   = -g
 EXTRA   = -O
-FFLAGS=  $(EXTRA)
+FFLAGS=  -ffixed-line-length-none $(EXTRA)
 CFLAGS = -Ae $(EXTRA)
-FC      = g77 $(FFLAGS)
+FC      = gfortran $(FFLAGS)
 CC      = cc $(CFLAGS)
 
 OBJECTS0 = AATSR.o ABSTRA.o AEROSO.o AKTOOL.o ATMREF.o AVHRR.o BBM.o BDM.o BRDFGRID.o \
Binary files 6SV1.1/sixsV1.1 and /Users/danclewley/Desktop/6SV1.1/sixsV1.1 differ
